libraries/lib-exceptions/AudacityException.h

```diff
@@ -218,15 +218,21 @@ noexcept(
 {
    try { return body(); }
    catch ( AudacityException &e ) {
-
+   #ifndef UNCAUGHT_EXCEPTIONS_UNAVAILABLE
+      const auto uncaughtExceptionsCount = std::uncaught_exceptions();
+   #endif  
       auto end = finally( [&]()
       noexcept(noexcept(
          std::function<void(AudacityException*)>{
             std::move(delayedHandler)} )) {
          // At this point, e is the "current" exception, but not "uncaught"
          // unless it was rethrown by handler.  handler might also throw some
          // other exception object.
+      #ifdef UNCAUGHT_EXCEPTIONS_UNAVAILABLE
          if (!std::uncaught_exception()) {
+      #else         
+         if (uncaughtExceptionsCount >= std::uncaught_exceptions()) {
```

Comment:
I think `==` is a sufficient test -- `uncaught_exceptions()` might have become more, but I don't know how it could become less!

But this is not incorrect so let it stand.

Reply:
Yeah, I was thinking to do `==`, but `>=` is even broader condition